elasticdl/python/elasticdl/layers/embedding.py

```diff
@@ -186,6 +189,17 @@ def _record_gradients(self, batch_embedding, ids):
         return batch_embedding
 
     def call(self, input):
+        if (
+            self.tape
+            and not tf.executing_eagerly()
+            and not self._embedding_and_ids_graph
+        ):
+            raise RuntimeError(
+                "Call `init_for_graph_mode` for each ElasticDL embedding"
```

Comment:
Maybe just init_for_graph_mode()

Reply:
Agree... a result of writing a lot of documents... :D